xml/System.Windows.Forms/DataGridViewComboBoxCell.xml

```diff
@@ -149,9 +149,16 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Creates a new <see cref="T:System.Windows.Forms.AccessibleObject" /> for this <see cref="T:System.Windows.Forms.DataGridViewComboBoxCell" /> instance.</summary>
+        <returns>A new <see cref="T:System.Windows.Forms.AccessibleObject" /> instance that supports the ControlType UIA property.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The new <xref:System.Windows.Forms.AccessibleObject> object is only available in applications that are recompiled to target versions of the .NET Framework starting with the .NET Framework 4.7.2. Applications that target an earlier version of the .NET Framework but are running on the .NET Framework 4.7.2 can opt in to this feature by using a compatibility switch. For additional information, see  [What's New in Accessibility in the .NET Framework](https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility).  
```

Comment:
This should be rephrased -- it implies that AccessibleObject itself is new. It's better to say something like, "The \<xref:System.Windows.Forms.AccessibleObject> returned by the \`CreateAccessibility\` method is only available..."

Reply:
I agree with Ron, word `new` is ambiguous,  but I have 2 additional points: method name is `CreateAccessibilityInstance`, and this method had been always available but it was inherited from the base class. in this release, we override this method and when the app opts in into the accessibility improvements, we return a object specific to the DataGridViewComboBoxCell, as opposed to object that works with any DataGridViewCell. Perhaps rephrase it like this:  "The <xref:System.Windows.Forms.AccessibleObject> returned by the `CreateAccessibilityInstance` method supports the `ControlType` UIA property only in applications that..."

Reply:
UIA stands for User Interface Automation @Tanya-Solyanik? I'm a bit confused on "the CreateAccessibilityInstance method supports the ControlType UIA property". (Note that I'm not the original author of this, I'm helping export from MSDN)

Reply:
Yes, you are right. usually it's called UI Automation, here is the link:
https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/ui-automation-overview